separated NFS server from Slurm playbook

hollysilk opened the pull request.

Commit:
separated NFS server from Slurm playbook

Commit:
changed Slurm var names to match new openhpc var names

priteau commented on ansible/nfs-infra.yml:
```diff
@@ -0,0 +1,30 @@
+  tasks:
+    - import_tasks: cluster-firewalld.yml 
+
+#- hosts: cluster
```
Can this commented block of code be removed since a similar version is now in `nfs.yml`?

hollysilk replied:
Yes, you're right, will remove.

priteau commented on ansible/group_vars/server:
```diff
@@ -0,0 +1,3 @@
+nfs_fstype: "xfs"
```
Shouldn't this file be named `nfs_server` rather than `server`?

hollysilk replied:
Groups `server` and `nfs_server` should both be created.

From jasmin-cluster-infra:
```
group_inventory_groups: 
      - cluster
      - "{{ [cluster_name, group.name] | join('_')}}"
      - "{{ group.name }}"
```

Is one preferable over the other?

priteau replied:
My thought was that, since this group_vars file potentially applies to all cluster deployed, naming it more specifically avoids confusion.

hollysilk replied:
Sounds reasonable to me. Will change.

Commit:
deleted commented block of code

Commit:
renamed nfs-server group_vars file

oneswig committed:
Some small changes for situations where the cluster names are not
the default values ('slurm' and 'nfs').  Added support for different
network resources and access via a single bastion host group that is common
to the deployment.

oneswig committed:
Merge branch 'nfs-server' of https://github.com/cedadev/jasmin-appliances into nfs-server

oneswig reviewed (approved):
Holly and I have both tested this and it appears to work as expected.
